src/plugins/plot/chart/MctChart.vue

```diff
@@ -202,9 +214,7 @@ export default {
     this.drawnOnce = false;
     const rootContainer = this.openmct.element;
     const options = {
-      root: rootContainer,
-      rootMargin: '0px',
-      threshold: 1.0
+      root: rootContainer
```

Comment:
we need otherwise defaults here  (specifically the `0` threshold) so our callback runs when the whole canvas is gone, or if a single pixel of the canvas is visible. this keeps the intersection conservative in its definition of "visible"